src/scripting/game_lua_kernel.cpp

```diff
@@ -1704,6 +1704,7 @@ int game_lua_kernel::impl_current_get(lua_State *L)
 	return_int_attrib("turn", play_controller_.turn());
 	return_string_attrib("synced_state", synced_state());
 	return_bool_attrib("user_can_invoke_commands", !events::commands_disabled && gamedata().phase() == game_data::TURN_PLAYING);
+	return_bool_attrib("is_replaying", play_controller_.is_replay());
```

Comment:
The convention for attributes of `wesnoth.current` is that reading it out makes sense as a phrase – "current map", "current schedule", "current event context", "current user can invoke commands", etc. This one doesn't fit that convention – "current is replaying" doesn't make sense.

Maybe `user_is_replaying`?

Reply:
Hmm… `wesnoth.scenario.is_replaying` doesn't really make sense… maybe you could put it in`wesnoth.interface` as a function…

Reply:
This still needs to be addressed.